src/20250319_Multi_FinanceStewardDeploymentPoolExposureModule/Values.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPriceOracleGetter} from 'aave-address-book/AaveV3.sol';
+
+library Values {
+  function getTokenAmountByDollarValue(
+    address underlying,
+    address oracle,
+    uint256 decimals,
+    uint256 dollarValue
+  ) internal view returns (uint256) {
+    dollarValue = decimals > 5 ? dollarValue : dollarValue * 10;
+    uint256 latestAnswer = IPriceOracleGetter(oracle).getAssetPrice(underlying);
+    return (dollarValue * 10 ** (8 + decimals)) / latestAnswer;
+  }
+
+  function getTokenAmountByDollarValueEthOracle(
+    address underlying,
+    address oracle,
+    uint256 decimals,
+    uint256 dollarValue,
+    uint256 ethPrice
+  ) internal view returns (uint256) {
+    dollarValue = decimals > 5 ? dollarValue : dollarValue * 10;
+    uint256 latestAnswer = IPriceOracleGetter(oracle).getAssetPrice(underlying);
+    uint256 tokenUsdPrice = (latestAnswer * ethPrice) / (10 ** 8);
```

Comment:
this approach of mixing eth & usd oracles is quite tricky, but I'm relatively certain this is wrong.
if it#s an eth oracle division must be by 1e18.

Perhaps good to add a test for both methods, assuring that the amount of usdc is roughly 1e6